garage/logger/tabular_input.py

```diff
@@ -0,0 +1,91 @@
+"""Contains the tabular class.
+
+This class allows the recording of tabular information for later output.
+TabularInput may be passed to the logger via its log() method.
+"""
+from contextlib import contextmanager
+
+import numpy as np
+import tabulate
+
+
+class TabularInput(object):
```

Comment:
No need to inherit from object in Python 3

Reply:
Fixed